src/track/beatgrid.cpp

```diff
@@ -3,28 +3,54 @@
 
 #include "track/beatgrid.h"
 #include "util/math.h"
+#include "maths/MathUtilities.h"
```

Comment:
Including files from qm-dsp outside of the Vamp plugin context is very confusing. Is it actually needed? Please remove this #include directive if not needed and try to use standard C++ features.